newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardAppearanceDialog.js

```diff
@@ -354,6 +373,56 @@ function LeaderboardAppearanceDialog({
               <LeaderboardPlaygroundCard />
             )}
             <Spacer />
+            <Text size="sub-title" noMargin>
+              <Trans>Visual appearance (advanced)</Trans>
+            </Text>
+            <Checkbox
+              label={<Trans>Use custom CSS for the leaderboard</Trans>}
+              disabled={(!useCustomCss && !canUseCustomCss) || isLoading}
```

Comment:
The checkbox is disabled if:
- it's loading (normal)
- of if you can't use custom css AND it's not checked yet.

In other words, this allows to disable the checkbox, but not enable it, in case you "lose" access to the subscription.

Reply:
Oh ok! Could you add a comment to explain that?

Reply:
Done 👍